types/spellchecker/index.d.ts

```diff
@@ -0,0 +1,63 @@
+// Type definitions for spellchecker 3.5
+// Project: http://atom.github.io/node-spellchecker
+// Definitions by: Dale Fenton <https://github.com/dalevfenton>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+export as namespace Spellchecker;
```

Comment:
It looks like this is to be used as a CommonJS module, so I don't think adding `Spellchecker` to the global scope is necessary. Is there a specific reason you might need to do this?

Reply:
The dts-gen had that bit in the file, I didn't really realize what it was doing. Will remove and update the PR, based on testing doesn't look like it's necessary.